<?xml version="1.0"?>
<launch>

    <rosparam file="$(find hsr_data_collection)/config/hsr_data_collection_config.yaml" command="load"/>

    <node name="rosbag_manager" pkg="hsr_data_collection" type="rosbag_manager.py" output="screen">
    </node>

    <node name="status_led_manager" pkg="hsr_data_collection" type="status_led_manager.py" output="screen">
    </node>

    <node name="dualshock_teleop_interface" pkg="hsr_data_collection" type="dualshock_interface.py" output="screen">
    </node>

</launch>
